<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1240</width>
    <height>810</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #F6F6F6;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>221</width>
      <height>741</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {
	background-color: #F8E8EE;
	border-radius: 20px solid;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="alarm_clock_Button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	color: rgb(65, 65, 65);
	border-radius: 20px solid;
}</string>
     </property>
     <property name="text">
      <string>Alarm Clock</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../GUI/PyQt5/images/icons/bonus/icons-24/alarm-clock-blue.png</normaloff>../../GUI/PyQt5/images/icons/bonus/icons-24/alarm-clock-blue.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="world_clock_Button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	color: rgb(65, 65, 65);
	border-radius: 20px solid;
}</string>
     </property>
     <property name="text">
      <string>World Clock</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>C:/Users/mahrayan/Downloads/world-map-flat-icon-free-vector.jpg</normaloff>C:/Users/mahrayan/Downloads/world-map-flat-icon-free-vector.jpg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="Pomodoro_Button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	color: rgb(65, 65, 65);
	border-radius: 20px solid;
}</string>
     </property>
     <property name="text">
      <string>Pomodoro</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>C:/Users/mahrayan/Downloads/14358941.png</normaloff>C:/Users/mahrayan/Downloads/14358941.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="alarm_label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(102, 102, 102);</string>
     </property>
     <property name="text">
      <string>Alarm Clock</string>
     </property>
    </widget>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>400</y>
      <width>931</width>
      <height>351</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>-5</number>
    </property>
    <property name="midLineWidth">
     <number>-5</number>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>929</width>
       <height>349</height>
      </rect>
     </property>
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>921</width>
        <height>351</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame {
	background-color: #F8E8EE;
	border-radius: 20px solid;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="user_alarm_label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(102, 102, 102);</string>
       </property>
       <property name="text">
        <string>Your Alarms</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="alarm_title_label">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>30</y>
      <width>311</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(102, 102, 102);</string>
    </property>
    <property name="text">
     <string>    Alarm Clock</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>130</y>
      <width>911</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
	background-color: #F8E8EE;
	border-radius : 20px solid,
}

</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="Create_new_alarm_Button">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>180</y>
       <width>261</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: #8785A2;
	color: rgb(255, 255, 255);
	border-radius: 20px solid;
}</string>
     </property>
     <property name="text">
      <string>Create New Alarm</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Quick_alarm_5_Button">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>180</y>
       <width>91</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: #F2BED1;
	border-radius: 20px solid;
}</string>
     </property>
     <property name="text">
      <string>+5m</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Quick_alarm_10_Button">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>180</y>
       <width>91</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: #F2BED1;
	border-radius: 20px solid;
}</string>
     </property>
     <property name="text">
      <string>+10m</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Quick_alarm_15_Button">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>180</y>
       <width>91</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: #F2BED1;
	border-radius: 20px solid;
}</string>
     </property>
     <property name="text">
      <string>+15m</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Quick_alarm_30_Button">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>180</y>
       <width>91</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: #F2BED1;
	border-radius: 20px solid;
}</string>
     </property>
     <property name="text">
      <string>+30m</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Quick_alarm_1h_Button">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>180</y>
       <width>91</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: #F2BED1;
	border-radius: 20px solid;
}</string>
     </property>
     <property name="text">
      <string>+1h</string>
     </property>
    </widget>
    <widget class="QLabel" name="Date">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>130</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>351</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
       <pointsize>36</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(65, 65, 65);</string>
     </property>
     <property name="text">
      <string>16:42:35</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1240</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
